libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -644,7 +644,6 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
       P2PKH: 28,
       P2SH: 85,
       XPUBVersion: 0x0488b21e,
-      hasTimestamp: true,
```

Comment:
I've only removed the variable here, even though those currencies are not supported by LL anymore, do you suggest I also remove those or should they be kept here @hzheng-ledger ?

Reply:
good catch. @Wozacosta I think we can remove it too to simplify our codebase. thanks a lot